Expensify/App
# Give Storybook a facelift

### Details
This adds our brand theme to our storybook docs. I am keeping this on hold in the meantime because I would like to use it to test https://github.com/Expensify/App/pull/6906.

### Fixed Issues
n/a

### Tests
1. Run `npm i && npm run storybook`
1. Verify that the documentation site builds and looks great!


### QA Steps
Go to https://staging.new.expensify.com/docs/index.html and verify that you can see the storybook docs with the themes matching the screenshots below.


### Tested On


### Screenshots
#### Web
<img width="1792" alt="Screen Shot 2022-01-12 at 3 55 42 AM" src="https://user-images.githubusercontent.com/47436092/149137075-b0454f33-f390-4bdf-aa41-690203bb87d8.png">
<img width="1792" alt="Screen Shot 2022-01-12 at 3 55 56 AM" src="https://user-images.githubusercontent.com/47436092/149137083-5255d161-4e5d-4441-b678-9fd3602124ae.png">
<img width="1792" alt="Screen Shot 2022-01-12 at 3 56 53 AM" src="https://user-images.githubusercontent.com/47436092/149137086-09dc677c-bc2d-4e69-a0c0-90ba94c99caa.png">
<img width="486" alt="Screen Shot 2022-01-12 at 3 56 23 AM" src="https://user-images.githubusercontent.com/47436092/149137089-ea2af419-257f-4bac-bf83-cd64fd225195.png">
<img width="1792" alt="Screen Shot 2022-01-12 at 3 56 09 AM" src="https://user-images.githubusercontent.com/47436092/149137090-134d7e74-ecfd-4dad-8fa4-9d448f01aadb.png">
